model/db.js

```diff
@@ -10,7 +10,7 @@ var Schema = mongoose.Schema;
 const memberSchema = new Schema({
     lastName: String,
     firstName: String,
-    skills: Array,
+    skills: [],
```

Comment:
You're getting an empty array because you've defined skills schema as an empty array. 
skills: Array or skills: {type: Array} is the correct way to define the schema. Please refer to mongoose docs http://mongoosejs.com/docs/guide.html